test/DeltaTriplesTest.cpp

```diff
@@ -139,9 +151,12 @@ TEST_F(DeltaTriplesTest, insertTriplesAndDeleteTriples) {
                                  [](const auto& item) { return item.first; });
   };
   auto UnorderedTriplesAre = [&mapKeys, this, &vocab, &localVocab](
+                                 auto t,
                                  const std::vector<std::string>& triples)
       -> testing::Matcher<const ad_utility::HashMap<
-          IdTriple<0>, DeltaTriples::LocatedTripleHandles>&> {
+          IdTriple<0>,
+          DeltaTriples::LocatedTripleHandles<decltype(t)::value>>&> {
+    (void)t;
```

Comment:
Can this be done with a template parameter? If not please give it a more meaningful name.

Reply:
Yes it could, but not in a lambda in C++17 :/